mon-pix/app/components/routes/campaigns/restricted/join-sco.hbs

```diff
@@ -57,13 +57,9 @@
   {{#if this.errorMessage}}
     <div class="join-restricted-campaign__error" aria-live="polite">{{{this.errorMessage}}}</div>
   {{/if}}
-  {{#if this.isLoading}}
-    <button type="button" disabled class="button button--big join-restricted-campaign__button"><span class="loader-in-button">&nbsp;</span></button>
-  {{else}}
-    <button type="submit" class="button button--big join-restricted-campaign__button" {{on 'click' this.submit}}>
-      {{t 'pages.join.button'}}
-    </button>
-  {{/if}}
+  <div class="join-restricted-campaign__actions">
+    <PixButton @type="submit" @triggerAction={{this.submit}}>{{t 'pages.join.button'}}</PixButton>
```

Comment:
**Suggestion** : <form onSubmit={{this.submit}}>

Reply:
En général, le mieux reste le on submit sur le form puisqu'il permet de faire la validation des champs avant le submit. Ce que ne permet pas l'action directement sur le bouton.
Je change ça du coup :)